src/main/scala/com/microsoft/partnercatalyst/fortis/spark/pipeline/BingPipeline.scala

```diff
@@ -15,9 +17,11 @@ object BingPipeline extends Pipeline {
 
   private def convertToSchema(stream: DStream[BingPost], transformContext: TransformContext): DStream[AnalyzedItem] = {
     stream.map(post => AnalyzedItem(
+      createdAtEpoch = now.getEpochSecond,
```

Comment:
They have a crawl-date but not a post-date ([BingPost.scala](https://github.com/CatalystCode/streaming-bing/blob/d1fdc2f4d5f69b052434c25429cc7ddc233eab38/src/main/scala/com/github/catalystcode/fortis/spark/streaming/bing/dto/BingPost.scala#L3-L8)). Not sure if that's something that would be more useful to us than a consistent timestamp.